storybook/src/elements/nuxeo-actions-menu/nuxeo-actions-menu.stories.js

```diff
@@ -4,18 +4,41 @@ import { html } from 'lit-html';
 import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-actions-menu';
 import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-link-button';
 
-storiesOf('UI/Widgets', module).addElement('nuxeo-actions-menu', ({ knobs }) => {
-  knobs();
+const iconsList = [
+  { icon: 'nuxeo:edit', label: 'Edit' },
+  { icon: 'nuxeo:attachment', label: 'Attachment' },
+  { icon: 'nuxeo:dashboard', label: 'Dashboard' },
+  { icon: 'nuxeo:add', label: 'Add' },
+  { icon: 'nuxeo:delete', label: 'Delete' },
+  { icon: 'nuxeo:download', label: 'Download' },
+  { icon: 'nuxeo:export', label: 'Export' },
+  { icon: 'nuxeo:edit', label: 'Edit' },
+  { icon: 'nuxeo:attachment', label: 'Attachment' },
+  { icon: 'nuxeo:dashboard', label: 'Dashboard' },
+];
+
+const label = 'Number of items';
+const defaultValue = 5;
+const options = {
+  range: true,
+  min: 1,
+  max: 10,
+  step: 1,
+};
```

Comment:
Cool approach.
Anyway, not sure if we want the user to add more than 10. It might be interesting to demonstrate how it behaves with lots of actions (scroll and so on).

Reply:
i changed it to 15. i chose a range so i can make icons look different and a be more realistic example. we can add more than 15 but i think that's a corner-case and not a good ux practice to start with